.gitignore

```diff
@@ -1,5 +1,7 @@
-
-#Build result
-
-*.log
-_pycache_/
\ No newline at end of file
+*/*.log
+/*.log
```

Comment:
wouldn't *.log take care of all log files in all directories?
Why do you need both of these?

Reply:
We don't need, i'll change that.